### DISCUSSION TITLE
Support Taiga, save tigers

### DISCUSSION THREAD

**Discussion by erengy** [erengy/taiga#1067] [General]
In the past couple of years, I've met some people who'd liked to donate to Taiga, whom I kindly refused. Recently I've been thinking of channeling their good will to something that matters, by encouraging them to make a donation to an appropriate cause instead.

I'm not sure if calling Taiga ["careware"](https://en.wikipedia.org/wiki/Careware) or ["charityware"](http://charityware.info/wiki/what-is-a-charityware) rather than "freeware" would make a positive impact, but I intend to set up a donation page on Taiga's website (something along the lines of the [Donate](http://erengy.com/donate) page on my personal website) and provide a link to it through the application.

I'm currently looking for charities that I can suggest in good conscience to Taiga users. To keep things somewhat relevant, I'm specifically looking for non-profit organizations that support wildlife, especially tigers. A quick search revealed about a dozen facilities and causes, but I have no means to verify their trustworthiness. Any help would be appreciated.

**Comment by Cael** [comment]
I know sites like www.charitynavigator.org are good resources for this. A quick search shows that Big Cat Rescue is trustworthy, but their focus is on rescuing abused animals. None of the ones on there seem to be very good for preservation, but I'll continue looking in to it, though the WWF is another good foundation.

**Comment by erengy** [comment]
Thanks for the information, @Cael. Someone else just told me about [21st Century Tiger](http://www.21stcenturytiger.org):

> It is a UK based foundation working under Zoological Society of London and they seem quite efficient. For instance, [here](http://www.21stcenturytiger.org/tiger-conservation/tiger-projects-indonesia/kerinci-seblat-tiger-protection-project/) they submit regular reports.

Apparently they're both members of the [International Tiger Coalition](https://en.wikipedia.org/wiki/International_Tiger_Coalition). ["Before You Donate"](http://www.sanctuaryfederation.org/gfas/information-tools-resources/before-you-donate/) page of Global Federation of Animal Sanctuaries also seems like a good resource on the topic.

**Comment by erengy** [comment]
A slightly-updated version of [taiga.erengy.com](http://taiga.erengy.com) is now live. I added an FAQ entry for donations as well.

**Comment by erengy** [comment]
I recently launched a [Patreon page](https://www.patreon.com/erengy). Upon reaching each goal, I will once donate the entire amount to an initiative saving the lives of actual tigers.
